packages/mux-viewer-count/src/index.ts

```diff
@@ -123,18 +132,39 @@ class MuxViewerCountElement extends globalThis.HTMLElement {
     this.setAttribute(Attributes.POLL_INTERVAL, `${value}`);
   }
 
+  enable() {
+    this.setAttribute('tabindex', '0');
+  }
+
+  disable() {
+    this.removeAttribute('tabindex');
+  }
+
   connectedCallback() {
+    if (!this.hasAttribute('disabled')) {
+      this.enable();
+    }
+
+    /** @TODO Update to use `viewerCount` prop when available (CJP) */
+    this.setAttribute('aria-label', toAriaLabel({}));
+    this.setAttribute('role', 'presentation');
```

Comment:
this can probably be in the constructor, but probably doesn't matter that much.

Reply:
We probably don't want to do that. Per spec:

- The element's attributes and children must not be inspected, as in the non-[upgrade](https://html.spec.whatwg.org/multipage/custom-elements.html#upgrades) case none will be present, and relying on upgrades makes the element less usable.
- The element must not gain any attributes or children, as this violates the expectations of consumers who use the [createElement](https://dom.spec.whatwg.org/#dom-document-createelement) or [createElementNS](https://dom.spec.whatwg.org/#dom-document-createelementns) methods.

\- From: https://html.spec.whatwg.org/multipage/custom-elements.html#custom-element-conformance

Reply:
oh, right, I always forget that. Eitherway. :shipit: